### DISCUSSION TITLE
launch geany in workspace 4 and move to that workspace

### DISCUSSION THREAD

**Discussion by gychang** [i3/i3#5858] [Q&A] [answered]
I am running i3 in puppylinux and trying to finalize my setup.  I have been trying with wmctrl to place geany in that workspace but without success.  Can this be done without the use of wmctrl?  Simple way of doing this in ~/.config/i3/config?

**Comment by okraits** [comment]
Assigning applications to specific workspaces is explained in the documentation: https://i3wm.org/docs/userguide.html#assign_workspace

**Comment by gychang** [comment]
thanks!, lots of information.  I  can't seem to find a how to get to the workspace i.e. where the geany is opened in one step.  i.e. without having to go to the workspace itself...

**Reply by okraits** [reply, marked as answer]
https://i3wm.org/docs/userguide.html#for_window

Example:

```
assign [class="Thunar"] → number 6
for_window [class="Thunar"] workspace number 6
```
